setup-script.py

```diff
@@ -168,7 +168,7 @@ def generate_manifests(template_vars: Dict[str, str], dest: Path) -> None:
             if file_or_dir.is_dir():
                 to_explore.append(file_or_dir)
             else:
-                dest_file = dest / file_or_dir.relative_to(K8_DIR)
+                dest_file = dest / file_or_dir.absolute().relative_to(K8_DIR.absolute())
                 dest_file.parent.mkdir(parents=True, exist_ok=True)
                 if str(file_or_dir).endswith(TEMPLATE_SUFFIX):
                     dest_file = dest_file.parent / (str(dest_file)[:-len(TEMPLATE_SUFFIX)] + '.yml')
```

Comment:
I had luck changing this line to 
```dest_file = (str(dest_file)[:-len(TEMPLATE_SUFFIX)] + '.yml')```

Reply:
You've found the right bug, but I think that fix doesn't work if `dest_file` isn't a relative path. I added a similar fix that I think works with absolute and relative paths.